### PR TITLE
Added line breaks

### LINKED ISSUE #247: rendering issue on small screens.
**Describe the bug**
The Text on the Roadmap goes outside the screen.

**Reproduce**
Visit the website on a mobile phone with a small device. Navigate to the roadmap.

**Fix**
Make the text continue on the next line.  

**Additional Comments**

### PR DESCRIPTION
**Pull request by ClientCrash**
**Description**
H1 tags are now multiline if the text is too long.

**Issue**
Closes #247 

**Additional Notes**
ewohgiuwegfiuewb

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -329,7 +329,24 @@ export default {
...
+}
+h3{
+  word-wrap: break-word;
+}
+h4{
+  word-wrap: break-word;
+}
+h5{
+  word-wrap: break-word;
+}
```
Comment (MarcelCoding) [inline pages/index.vue:347]:
```suggestion
h1, h2, h3, h4, h5, h6 {
  word-wrap: break-word;
}
```

Comment (MarcelCoding) [reply pages/index.vue:347]: And maybe put this into the `assets/css/global.scss`.

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

File: pages/index.vue [hunk 2]
```diff
@@ -329,7 +329,9 @@ export default {
   }
 }
 </script>
+<style lang="scss">
 
+</style>
```
Comment (MarcelCoding) [inline pages/index.vue:334]:
```suggestion

```

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by MarcelCoding** [comment]
Cloud you add this also to buttons (maybe also other elements where the error occurs). (#248) Because this would be a better solution.

File: assets/css/global.scss [hunk 3]
```diff
@@ -2,3 +2,6 @@
 body {
   font-family: Alata, sans-serif;
 }
+h1, h2, h3, h4, h5, h6,button,article,b,input[type=submit] {
```
Comment (MarcelCoding) [inline assets/css/global.scss:5]:
```suggestion
h1, h2, h3, h4, h5, h6, button, article, b, input[type=submit] {
```

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Review by JannikEmmerich — APPROVED** [review]

**Review by MarcelCoding — APPROVED** [review]

**Pull request merged**